AlphaWallet/Tokens/Coordinators/TokensCoordinator.swift

```diff
@@ -188,6 +188,12 @@ extension UIPasteboard {
 
 extension TokensCoordinator: TokensViewControllerDelegate {
 
+    func buyCryptoSelected(in viewController: UIViewController) {
+        guard config.enabledServers.contains(.main) else { return }
```

Comment:
Shall we move this condition to control whether the buy button is shown?

Reply:
checking